runscripts/awicm-pism/coupling/env_pism.py

```diff
@@ -39,7 +38,9 @@ def prepare_environment(config):
             "DEBM_EXE": config["pism"]["debm_path"],
             "MY_OBLIQUITY": config["pism"]["debm_obl"], 
             "iterative_coupling_atmosphere_pism_regrid_method": "DOWNSCALE", 
-            #"PISM_OCEAN_PICO_BASINS_FILE": "/home/ollie/lackerma/pool_pism/basins/antarctica.16km.nc",
+            #fesom
+            "iter_coup_interact_method_oce2ice": config["pism"].get("oce2ice_method", "OCEANTEMPSALT"),
+            "PISM_OCEAN_PICO_BASINS_FILE": config["pism"].get("basin_file", "/home/ollie/lackerma/pool_pism/basins/antarctica.16km.nc"),
```

Comment:
We probably want to control this with a variable, so that we can set it up automatically later in other computers, but we can do that later on.